pt-BR/active_record_callbacks.md

```diff
@@ -400,12 +400,12 @@ end
 
 You can declare as many callbacks as you want inside your callback classes.
 
-Transaction Callbacks
+Callbacks de Transação
```

Comment:
```suggestion
*Callbacks* de Transação
```

Acho que podemos manter o itálico no título também, o que acha?

Reply:
Sim, concordo :)